### ISSUE TITLE
Back-gesture triggers TabBars to scroll

### ISSUE THREAD

**Issue by Sesa1988** [flutter/flutter#121070]
Hi,

Its just a small thing but when you use the back-gesture to navigate to the previous screen, it leads TabBars and other scrollable widgets to switch pages, what is not the case on native android apps.

**Comment by exaby73** [comment]
Hello @Sesa1988. Thank you for filing this issue. Can you please provide the following information?

- A [complete and minimal, reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example). Ideally, a single `main.dart` file that can be run directly.
- The output of `flutter doctor -v`

Please provide the information in the form of text. See how to make collapsible sections with Markdown [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections).

**Comment by Sesa1988** [comment]
> i have same issue.
> When i touch pageview/tabbar(some widget with Horizontal Scroller) edge to navigate to the previous screen, Cupertino Back Gesture does not work in Android Device

Thats not the same issue since it navigates back in my case but should not.

**Comment by exaby73** [comment]
@Sesa1988 Were you able to make a [complete and minimal, reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example)?

**Comment by Sesa1988** [comment]
> @Sesa1988 Were you able to make a [complete and minimal, reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example)?

Hi @exaby73. Here is the repository with a reproduction. It can be reproduced on both, a simulator or a real device but its way easier to reproduce on a hardware device.

https://github.com/Sesa1988/flutter_navigateback_contentscroll_issue


https://user-images.githubusercontent.com/26410357/224117151-3d95eb24-caf6-4589-9f91-efcec620102f.mp4

**Comment by exaby73** [comment]
Could you please post your output of `flutter doctor -v`?

**Comment by Sesa1988** [comment]
> Could you please post your output of `flutter doctor -v`?

[!] Flutter (Channel stable, 3.7.6, on macOS 13.0 22A380 darwin-x64, locale en-DE)
    • Flutter version 3.7.6 on channel stable at /Users/user/Development/flutter
    ! Warning: `flutter` on your path resolves to /Users/user/Development/flutter/bin/flutter, which is not
      inside your current Flutter SDK checkout at /Users/user/Development/flutter. Consider adding
      /Users/user/Development/flutter/bin to the front of your path.
    ! Warning: `dart` on your path resolves to /Users/user/Development/flutter/bin/dart, which is not
      inside your current Flutter SDK checkout at /Users/user/Development/flutter. Consider adding
      /Users/user/Development/flutter/bin to the front of your path.
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 12cb4eb7a0 (9 days ago), 2023-03-01 10:29:26 -0800
    • Engine revision ada363ee93
    • Dart version 2.19.3
    • DevTools version 2.20.1
    • If those were intentional, you can disregard the above warnings; however it is recommended to use "git"
      directly to perform update checks and upgrades.

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
    • Android SDK at /Users/user/Library/Android/sdk
    • Platform android-33, build-tools 33.0.1
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14B47b
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[✓] VS Code (version 1.76.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.60.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 13.0 22A380 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 110.0.5481.177

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.

**Comment by exaby73** [comment]
I am unable to reproduce this issue on any emulator, or my Samsung Galaxy S9+. Could you try running on master channel and see if you can still reproduce this issue?

**Comment by exaby73** [comment]
@Sesa1988 Yes but it could already be fixed in the master branch. I cannot reproduce it so I cannot verify if it's fixed or not. Please do try on the master branch
